README.md

```diff
@@ -1,2 +1,203 @@
-# Ruby_Capstone-_Project-
-My partner, @carlosz96, and I collaborated on a console app for this project. It assists us in maintaining a comprehensive inventory of our possessions, including books, music albums, movies, and games.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Catalog Of My Things</b></h3>
+</div>
+
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [🚀 Video Link](#video-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of my Things <a name="about-project"></a>
+
+**Catalog of my Things** In this project you will learn how to use a module inside your class.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
+<details>
+<summary>Rspec</summary>
+  <ul>
+    <li><a href="https://rspec.info/">Rspec</a></li>
+  </ul>
+</details>
+
+## 🚀 Video Demo <a name="video-demo"></a>
+
+[Video Presentation](https://www.loom.com/share/63eefe0879174ddb891b48e650aa394b)
```

Comment:
@peterrobert man for real? you spent almost 2hrs to provide this kind of feedback?

Reply:
Hi @peterrobert, Thank you for the review, we are sorry, we have problems recording and making both voices heard, the program works correctly, is this strictly necessary? Taking into account how late it is in our countries, it is difficult for us to record another video, now please help us with this.

Reply:
- Hey @jlberbesi, As requested by the previous reviewer, your partner needs to be in the video as this is the general workflow used in a pair-programming capstone project. You mentioned that your partner couldn't join you in the previous video, and also, you've now stated the reason they can't join you in the video :muscle: Regarding the second video you've added, I just want to point out a few things you should consider:
*Please, kindly put into consideration these few pointers:*

   - In this, you're angry and yelling, this is not professional, having to know that some recruiter might see this video, and this means you don't have good conflict resolution skills. It is fine to be angry, but let's try to keep our emotions in check when making videos like this.
   - You claimed the reviewer is blocking you, however, that's the requirement of making the video. Also, you're not being polite making reference to him. Please, let's be mindful of this.
   -

Reply:
- Hey @jlberbesi, As requested by the previous reviewer, your partner needs to be in the video as this is the general workflow used in a pair-programming capstone project. You mentioned that your partner couldn't join you in the previous video, and also, you've now stated the reason they can't join you in the video :muscle: Regarding the second video you've added, I just want to point out a few things you should consider:
*Please, kindly take into consideration these few pointers:*

   - In this, you're angry and yelling, this is not professional, having to know that some recruiter might see this video, and this means you don't have good conflict resolution skills. It is fine to be angry, but let's try to keep our emotions in check when making videos like this.
   - You claimed the reviewer is blocking you, however, that's the requirement of making the video. Also, you're not being polite referring to him. Please, let's be mindful of this.

---
   
   > To summarize it all, even if I'm meant to approve your project, with the way and manner you've presented this video, I'm sorry you'd have to record another video even though your partner can't join the video at this moment. Be polite as much as you can, then your project can be approved.

Reply:
@CarlosZ96 and @jlberbesi that was very unprofessional of you. I spent less than an hour reviewing your project. The reason why your project took so long is that before I took it for review, it was on the dashboard since it's a Sunday morning and there were no reviewers. Also, the video presentation is a requirement for this project. Kindly if you need your project reviewed in time, kindly submit your project earlier for review.